sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml

```diff
@@ -34,7 +34,7 @@ fields:
 - description:
   mode: NULLABLE
   name: job_id
-  type: TIMESTAMP
+  type: INTEGER
```

Comment:
This change is not backwards compatible, how are we planning to handle that?

Reply:
I have already recreated the table. This is fine since the table is overwritten by the corresponding ETL anyways.